addons/binding/org.openhab.binding.dsmr/OSGI-INF/org.openhab.binding.dsmr.internal.DSMRHandlerFactory.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the service declaration files.

Reply:
Or at least move it. This binding could still not be ported to DS annotations and those files are actually required then

Reply:
Both files do not belong here. They are located in a different binding location: `org.openhab.binding.dsmr`.

Reply:
Just remove. These are left over because the binding migration was merged. He did a rebase and then these files remain because they were not part of the repository.

Reply:
True, @hilbrand. I‘ll remove them tomorrow.